codice/ddf
# DDF-4446: Added feedback when home is set

#### What does this PR do?
Gives user feedback when home location is set.
#### Who is reviewing it? 
@brianfelix 
@Corey-Collins 
@jrnorth 
#### Select relevant component teams: 
@codice/ui 
#### Ask 2 committers to review/merge the PR and tag them here.
@bdeining 
@andrewkfiedler 
#### How should this be tested?
1. Open search ui map
2. Click `Set Home` control
3. Observe announcement that new home has been set
#### Any background context you want to provide?
#### What are the relevant tickets?
[DDF-4446](https://codice.atlassian.net/browse/DDF-4446)
#### Screenshots
<img width="1792" alt="screen shot 2019-01-15 at 8 45 31 am" src="https://user-images.githubusercontent.com/118054/51184334-f9b99080-18a1-11e9-8977-6bbc8a1fe570.png">
#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
